srcs/server/Server.cpp

```diff
@@ -20,6 +20,10 @@ Server::~Server() {
 		delete h_it->second;
 	}
 	SyscallWrap::closeWr(listen_sd_ DEBUG_INFO);
+	Server::serverSettingsMap::iterator s_it;
```

Comment:
Don't you have to initialize s_it to something like  .begin() ?

Reply:
Good catch! Thank you!